go-controller/pkg/ovn/ovn_test.go

```diff
@@ -564,7 +566,19 @@ func (o *FakeOVN) NewSecondaryNetworkController(netattachdef *nettypes.NetworkAt
 			secondaryController = &l2Controller.BaseSecondaryNetworkController
 			o.fullSecondaryL2Controllers[netName] = l2Controller
 		case types.LocalnetTopology:
-			localnetController := NewSecondaryLocalnetNetworkController(cnci, nInfo, o.networkManager.Interface())
+			testDB := []libovsdbtest.TestData{
+				&vswitchd.OpenvSwitch{
+					ExternalIDs: o.externalIDS,
+				},
+			}
+			dbSetup := libovsdbtest.TestSetup{
+				OVSData: testDB,
+			}
+			ovsClient, _, err := libovsdbtest.NewOVSTestHarness(dbSetup)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
maybe move this into an helper to hide away some of this complexity ? 

Something like `setupOVSWithExternalIDs` or similar.

nit: you don't actually need the `testDB` variable, do you ? You can just inline it below, right ?